### ISSUE TITLE
USD 25.8: Accessibility Doc doesn't mention purpose

### ISSUE THREAD

**Issue by dgovil** [PixarAnimationStudios/USD#3726]
@dsyu-pixar  for your reference, I think this should be `AccessibilityAPI::default` as outlined here https://github.com/PixarAnimationStudios/OpenUSD/pull/3271 since its a multiple apply schema?

https://github.com/PixarAnimationStudios/OpenUSD/blob/8e11e081cdc75acf8dc07bdef9418d9622835df9/pxr/usd/usdUI/userDoc/overview.md?plain=1#L122

**Comment by jesschimein** [comment]
Filed as internal issue #USD-11228

(This is an automated message. See [here](https://openusd.org/release/contributing_to_usd.html#github-issues) for more information.)

**Comment by dsyu-pixar** [comment]
@dgovil thank you for catching this! As the example is showing use of both "accessibility:default" and "accessibility:size", we'll correct the prepend to be: `prepend apiSchemas = ["AccessibilityAPI:default", "AccessibilityAPI:size"]`

**Comment by dgovil** [comment]
Awesome, thank you!
